tests/static/commcareConnect/learn_module.xml

```diff
@@ -16,6 +16,7 @@
 				</data>
 			</instance>
 			<bind vellum:nodeset="#form/name" nodeset="/data/name" type="xsd:string" />
+			<bind vellum:nodeset="#form/module_1" nodeset="/data/module_1" />
```

Comment:
Would it be worth adding a `relevant` attribute here (and also in the test)? It would be more consistent with the other question type tests in this module.

Reply:
https://github.com/dimagi/Vellum/pull/1098/commits/88c4bcb49726ca7f5794bb460b19b20a1921c47e